### PR TITLE
Change the type for the Aspect field of each Site from String to Number.

### PR DESCRIPTION
**Pull request by Baalmart**
# Change the type for the Aspect field of each Site from String to Number.

**_WHAT DOES THIS PR DO?_**
Change the type for the aspect field of each Site from String to Number.

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
[PLAT-1069]

**_HOW DO I TEST OUT THIS PR?_**
README

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [x] [Create new Site](https://app.gitbook.com/o/-MCogOVGQqjUharJKchL/s/-MKqyQkcfs54GYi-q96G/device-registry/create-sites#create-site)
- [x] [Update Site](https://app.gitbook.com/o/-MCogOVGQqjUharJKchL/s/-MKqyQkcfs54GYi-q96G/device-registry/create-sites#update-site)

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

N/A

**_ARE THERE ANY RELATED PRs?_**

N/A

[PLAT-1069]: https://airqoteam.atlassian.net/browse/PLAT-1069?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by 123MwanjeMike — APPROVED** [review]
LGTM

**Review by NoahNsimbe — APPROVED** [review]
LGTM

File: src/device-registry/models/Site.js [hunk 1]
```diff
@@ -431,7 +430,6 @@ siteSchema.statics = {
           "devices.tags": 0,
           "devices.description": 0,
           "devices.isUsedForCollocation": 0,
-          "devices.createdAt": 0,
```
Comment (Baalmart) [inline src/device-registry/models/Site.js:434]: enabled the createdAt field when getting device details with the get Sites endpoint. cc: @ja-odur

File: src/device-registry/models/Site.js [hunk 2]
```diff
@@ -414,7 +414,6 @@ siteSchema.statics = {
           "airqlouds.location": 0,
           "airqlouds.airqloud_tags": 0,
           "airqlouds.long_name": 0,
-          "airqlouds.createdAt": 0,
```
Comment (Baalmart) [inline src/device-registry/models/Site.js:417]: enabled the createdAt field when getting Airqloud details with the get Sites endpoint. cc: @ja-odur

File: src/device-registry/models/Site.js [hunk 3]
```diff
@@ -121,7 +121,7 @@ const siteSchema = new Schema(
       type: String,
     },
     aspect: {
-      type: String,
+      type: Number,
```
Comment (Baalmart) [inline src/device-registry/models/Site.js:124]: change type from String to Number.

**Pull request merged**